### ISSUE TITLE
Cant Save in Tiled

### ISSUE THREAD

**Issue by paxstalkers** [bjorn/tiled#1259]
Hi, when i attempt to save in Tiled i get the "Tiled stopped responding message" Any ideas whats going on?

**Comment by bjorn** [comment]
Does it matter what you're trying to save? Did you try saving just an empty map?

Which version of Tiled are you using, and did you use a version of Tiled before that did work?

**Comment by paxstalkers** [comment]
Yeah I've tried saving both an empty and a full map. I get the same error twice

0.16 also no this is my first time using

**Comment by bjorn** [comment]
Hmm, I've never heard of this problem before. Could you try whether you have the same problem with older releases at https://github.com/bjorn/tiled/releases ? Also, are you using the 32-bit or 64-bit version?

**Comment by paxstalkers** [comment]
Will try, thanks for getting back to me so quick. I usually have a talent for making things that work perfectly stop working haha

64-bit version. I have double checked and made sure my laptop is the same

**Comment by bjorn** [comment]
> Will try, thanks for getting back to me so quick.

You're lucky that I'm having my Tiled development day right now. :-)

Also I desperately need more information to have any chance of fixing this issue. The 0.16 release has been downloaded over 15000 times so in general I'm quite sure that saving shouldn't be crashing.

Maybe you could try whether the destination folder in which you're saving matters?

**Comment by paxstalkers** [comment]
Hey sorry for the delay, I downloaded past 64 versions and none worked. I've redownloaded as a 32 bit version and seems to be working fine now. But cant explain why it doesnt save with a 64bit version. I have a 64 bit operating system so not too sure.

Anyway thanks for everything. Tiled rocks

**Comment by Seanba** [comment]
pax, out of curiosity, what is the absolute path of the file you are trying to save to? I wonder if there is something "special" about that directory + filename that Tiled isn't taking into account?
